bring_api/bring.py

```diff
@@ -694,3 +759,312 @@ async def does_user_exist(self, mail: Optional[str] = None) -> bool:
+                async with self._session.get(url) as r:
+                    _LOGGER.debug("Response from %s: %s", url, r.status)
+                    r.raise_for_status()
+
+                    try:
+                        self.__translations[locale] = await r.json()
+                    except JSONDecodeError as e:
+                        _LOGGER.error(
+                            "Exception: Cannot load articles.%s.json:\n%s",
+                            locale,
+                            traceback.format_exc(),
+                        )
+                        raise BringParseException(
+                            f"Loading article translations for locale {locale}"
+                            "failed during parsing of request response."
+                        ) from e
+            except asyncio.TimeoutError as e:
+                _LOGGER.error(
+                    "Exception: Cannot load articles.%s.json::\n%s",
+                    locale,
+                    traceback.format_exc(),
+                )
+                raise BringRequestException(
+                    f"Loading article translations for locale {locale}"
+                    "failed due to connection timeout."
+                ) from e
+
+            except aiohttp.ClientError as e:
+                _LOGGER.error(
+                    "Exception: Cannot load articles.%s.json:\n%s",
+                    locale,
+                    traceback.format_exc(),
+                )
+                raise BringRequestException(
+                    f"Loading article translations for locale {locale}"
+                    "failed due to request exception."
+                ) from e
+
+    def __translate(
+        self,
+        item_id: str,
+        *,
+        to_locale: Optional[str] = None,
+        from_locale: Optional[str] = None,
+    ) -> str:
+        """Translate a catalog item from or to a language.
+
+        Parameters
+        ----------
+        item_id : str
+            Item name.
+        to_locale : str
+            locale to translate to.
+        from_locale : str
+            locale to translate from.
+
+        Returns
+        -------
+        str
+            Translated Item name.
+
+        Raises
+        ------
+        BringTranslationException
+            If the translation fails.
+
+        """
+
+        locale = to_locale or from_locale
+
+        if locale == "de-CH":
+            return item_id
+
+        if not locale:
+            _LOGGER.debug("One of the arguments from_locale or to_locale required.")
+            raise ValueError("One of the arguments from_locale or to_locale required.")
+
+        if locale not in self.supported_locales:
+            _LOGGER.debug("Locale %s not supported by Bring.", locale)
+            raise ValueError(f"Locale {locale} not supported by Bring.")
+        try:
+            if to_locale:
+                item_id = self.__translations[locale].get(item_id, item_id)

```

Comment:
Can be done as onliner: `item_id = self.__translations[locale].get(item_id, item_id) if to_locale else item_id = ({value: key for key, value in self.__translations[locale].items()}).get(item_id, item_id)`